packages/client/src/client/App/Credit/CreditRfqs/CreditRfqCards/Quote/styled.ts

```diff
@@ -65,42 +68,52 @@ function getBuySellHighlightTextColor(theme: Theme, direction: Direction) {
   ]
 }
 
-export const QuoteDot = styled.div<CommonProps>`
+export const QuoteDot = styled.div<AnimatedRowProps>`
   height: 4px;
   width: 4px;
   border-radius: 4px;
-  background-color: ${({ theme, highlight, direction }) =>
+  background-color: ${({ theme, highlight, direction, passed }) =>
     highlight
       ? theme.textColor
+      : passed
+      ? `#686D74`
```

Comment:
I assumed the previous person who worked on this used strings because it wasn't in our palette. But this one in particular seems to be in. I'll check all the color literals again.

Reply:
Changed to use color in palette